.studio/sample-data-service

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Helper methods specific for the sample-data-service
+
+document "start_sample_data_service" <<DOC
+  Build and start the local sample-data-service
```

Comment:
Let's be consistent. Here you use "sample-data-service" and on the next function you use "Sample data service".

Reply:
thanks for nitpick suggestion!